yb-voyager/src/srcdb/postgres.go

```diff
@@ -251,3 +250,25 @@ func (pg *PostgreSQL) GetCharset() (string, error) {
 func (pg *PostgreSQL) FilterUnsupportedTables(tableList []*sqlname.SourceName) []*sqlname.SourceName {
 	return tableList
 }
+
+func (pg *PostgreSQL) FilterEmptyTables(tableList []*sqlname.SourceName) []*sqlname.SourceName {
+	nonEmptyTableList := make([]*sqlname.SourceName, 0)
+	for _, tableName := range tableList {
+		query := fmt.Sprintf(`SELECT false FROM %s.%s LIMIT 1;`, tableName.SchemaName.Unquoted, tableName.ObjectName.MinQuoted)
```

Comment:
There is `tableName.Qualified.MinQuoted` that can be used instead of the `%s.%s`. Fix all such occurrences.